pkg/generators/system/config/options.go

```diff
@@ -94,8 +94,8 @@ func NewOptions(spec saasv1alpha1.SystemSpec) Options {
 		RailsLogLevel:    &pod.ClearTextValue{Value: *spec.Config.Rails.LogLevel},
 		RailsLogToStdout: &pod.ClearTextValue{Value: "true"},
 
-		SphinxAddress: &pod.ClearTextValue{Value: SystemSphinxServiceName},
-		SphinxPort:    &pod.ClearTextValue{Value: fmt.Sprintf("%d", *spec.Sphinx.Config.Thinking.Port)},
+		SearchServerAddress: &pod.ClearTextValue{Value: *spec.Config.SearchServer.Host},
+		SearchServerPort:    &pod.ClearTextValue{Value: fmt.Sprintf("%d", *spec.Config.SearchServer.Port)},
```

Comment:
I think I'm not getting this. This configures the FTS engine in system to point to `spec.config.searchServer`, but `spec.config.searchServer` is also used as the Service name of the searchd statefulset. How do you start the searchd statefulset while system still uses the old sphinx one?

Reply:
The service name is defined on the Sphinx and Searchd config, using the `ServiceName` attribute. This is only used to set the environment variable.